coralproject/talk
# [CORL-1629] SSO Fix

## What does this PR do?

Some users could have been imported from a previous version of Coral where the `lastIssuedAt` is not set on the SSO profile. This marks the field as optional and treats it more cautiously.



## What changes to the GraphQL/Database Schema does this PR introduce?

- Marks the `lastIssuedAt` on the `SSOProfile` as optional.


## How do I test this PR?


With a user that doesn't have the `lastIssuedAt` value on a SSO profile, submit a SSO token with a `iat` and new email address and see the change!
